src/CV8ScriptRuntime.h

```diff
@@ -127,4 +127,37 @@ class CV8ScriptRuntime : public alt::IScriptRuntime
 			return result;
 		}
 	};
+
+	using ResourcesLoadedCallback = void (*)(v8::Local<v8::ScriptOrModule> referrer, v8::Local<v8::String> specifier, const void* promise);
+
+	class ResourcesLoadedResult
+	{
+	public:
+		ResourcesLoadedResult(v8::Local<v8::ScriptOrModule> referrer, v8::Local<v8::String> specifier, const void* promise, ResourcesLoadedCallback cb)
+		{
+			_referrer = referrer;
+			_specifier = specifier;
+			_promise = promise;
+			_callback = cb;
+		}
+
+		v8::Local<v8::ScriptOrModule> _referrer;
+		v8::Local<v8::String> _specifier;
+		ResourcesLoadedCallback _callback;
+		const void* _promise;
+
+		void call()
+		{
+			if (&_promise == nullptr) return;
```

Comment:
What did you intend to do here with getting address of pointer `_promise` and comparing it against null pointer keyword?

Reply:
Leftover code from debugging, removed it